packages/contracts-bedrock/test/Safe/DeployOwnership.t.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    DeployOwnership,
+    SafeConfig,
+    SecurityCouncilConfig,
+    DeputyGuardianModuleConfig,
+    LivenessModuleConfig
+} from "scripts/DeployOwnership.s.sol";
+import { Test } from "forge-std/Test.sol";
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { ModuleManager } from "safe-contracts/base/ModuleManager.sol";
+import { GuardManager } from "safe-contracts/base/GuardManager.sol";
+
+import { LivenessGuard } from "src/Safe/LivenessGuard.sol";
+import { LivenessModule } from "src/Safe/LivenessModule.sol";
+import { DeputyGuardianModule } from "src/Safe/DeputyGuardianModule.sol";
+import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
+
+contract DeployOwnershipTest is Test, DeployOwnership {
+    address internal constant SENTINEL_MODULES = address(0x1);
+    // keccak256("guard_manager.guard.address")
+    bytes32 internal constant GUARD_STORAGE_SLOT = 0x4a204f620c8c5ccdca3fd54d003badd85ba500436a431f0cbda4f558c93c34c8;
+
+    function setUp() public override {
+        super.setUp();
+        run();
```

Comment:
nit on namespacing this also so it's clear where this is from
```suggestion
        Deploy.run();
```

For some reason locally I'm getting `Failed to install Solc 0.8.25: Unknown version provided` so I have not verified this compiles, but it should

Reply:
For reasons unknown to me, both `DeployOwnership.run()` nor `Deploy.run()` give me

```
Error (9582): Member "run" not found or not visible after argument-dependent lookup in type(contract DeployOwnership).
```

We've agreed out of band that this is OK